base/sparse/sparsematrix.jl

```diff
@@ -2268,22 +2268,25 @@ function spset!{Tv,Ti<:Integer}(A::SparseMatrixCSC{Tv}, x::Tv, I::AbstractVector
     ((I[end] > m) || (J[end] > n)) && throw(DimensionMismatch(""))
     nnzA = nnz(A) + lenI * length(J)
 
-    colptr = A.colptr
     rowvalA = rowval = A.rowval
     nzvalA = nzval = A.nzval
 
     rowidx = 1
     nadd = 0
-    @inbounds for col in 1:n
-        rrange = colptr[col]:(colptr[col+1]-1)
-        (nadd > 0) && (colptr[col] = colptr[col] + nadd)
+    for col in 1:n
```

Comment:
You mean w.r.t. @nanosoldier? It pulls down the merge commit of the PR as soon as the job runs, so force pushing after it's started won't have any effect on the current job. The easiest thing for us to do here is just submit the new job whenever the PR is ready (currently, there's not a good way to cancel jobs without manually hacking around the submission system).